### PR TITLE
faq: clarify which reference to use when citing RIOT

### LINKED ISSUE #18: Academic citation
It is not crystal clear what paper to cite, should someone cite RIOT in an academic context.
How about adding in the landing page's FAQ a question like:" I want to cite RIOT, which reference should I use?"
and indicate in the answer the IEEE IoT Journal ref from 2018?

Emmanuel Baccelli, Cenk Gündogan, Oliver Hahm, Peter Kietzmann, Martine Lenders, Hauke Petersen, Kaspar Schleiser, Thomas C. Schmidt, Matthias Wählisch, RIOT: An Open Source Operating System for Low-End Embedded Devices in the IoT, IEEE Internet of Things Journal, Vol. 5, No. 6, pp. 4428-4440, December 2018.

### PR DESCRIPTION
**Pull request by waehlisch**
This PR fixes #18.

### CODE REVIEW

File: _faqs/faq-12.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+question: I want to cite RIOT. Which reference should I use?
+---
+
+Please consider the following references for citation.
+
+  * Emmanuel Baccelli, Cenk G&uuml;ndogan, Oliver Hahm, Peter Kietzmann, Martine Lenders, Hauke Petersen, Kaspar Schleiser, Thomas C. Schmidt, Matthias W&auml;hlisch, <a href="{{ "assets/pdf/riot-ieeeiotjournal-2018.pdf" | relative_url }}">RIOT: An Open Source Operating System for Low-End Embedded Devices in the IoT</a>, IEEE Internet of Things Journal, Vol. 5, No. 6, pp. 4428-4440, December 2018.
```
Comment (chrysn) [inline _faqs/faq-12.md:7]: New to the website so asking the stupid question: Do we really have to escape things down to ASCII? (If so, I think it's G&uuml;ndo&gbreve;an; if not, Gündoğan).

Examples of non-ASCII characters in existing (and working) text are _data/riot_boards.yml (TRÅDFRI) and the various names in community.html.

(There's nothing wrong per se with it in this case, I'm more trying not to get us take up habits that others later imitate and needlessly overencode.)

Comment (waehlisch) [reply _faqs/faq-12.md]: thanks for spotting. copy&paste failure. fixed.

> Do we really have to escape things down to ASCII? 
>
we don't need to but we allow.

File: _faqs/faq-12.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+question: I want to cite RIOT. Which reference should I use?
+---
+
+Please consider the following references for citation.
+
+  * Emmanuel Baccelli, Cenk G&uuml;ndo&gbreve;an, Oliver Hahm, Peter Kietzmann, Martine Lenders, Hauke Petersen, Kaspar Schleiser, Thomas C. Schmidt, Matthias W&auml;hlisch, <a href="{{ "assets/pdf/riot-ieeeiotjournal-2018.pdf" | relative_url }}">RIOT: An Open Source Operating System for Low-End Embedded Devices in the IoT</a>, IEEE Internet of Things Journal, Vol. 5, No. 6, pp. 4428-4440, December 2018.
```
Comment (leandrolanzieri) [inline _faqs/faq-12.md:7]: `&gbreve;` is not rendering correctly for me
error_faq

Comment (leandrolanzieri) [reply _faqs/faq-12.md:7]: On the other hand, using ğ directly works

Comment (waehlisch) [reply _faqs/faq-12.md]: just pushed a fix.

Comment (leandrolanzieri) [reply _faqs/faq-12.md]: That solves it

File: _faqs/faq-12.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+question: I want to cite RIOT. Which reference should I use?
+---
+
+Please consider the following references for citation.
+
+  * Emmanuel Baccelli, Cenk G&uuml;ndo&#287;an, Oliver Hahm, Peter Kietzmann, Martine Lenders, Hauke Petersen, Kaspar Schleiser, Thomas C. Schmidt, Matthias W&auml;hlisch, <a href="{{ "assets/pdf/riot-ieeeiotjournal-2018.pdf" | relative_url }}">RIOT: An Open Source Operating System for Low-End Embedded Devices in the IoT</a>, IEEE Internet of Things Journal, Vol. 5, No. 6, pp. 4428-4440, December 2018.
```
Comment (leandrolanzieri) [inline _faqs/faq-12.md:7]: The link to the document is not correct, it should be assets/pdf**s**...

Comment (waehlisch) [reply _faqs/faq-12.md]: arg. Jekyll didn't update locally. thanks!. fixed.

**Review by leandrolanzieri — APPROVED** [review]
ACK

**Pull request merged**